frontend/src/pages/Register.jsx

```diff
@@ -1,7 +1,206 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { Grid } from "@mui/material";
+import axios from "axios";
+import { toast } from "react-toastify";
+import Input from "../components/Input";
+
 
 const Register = () => {
-  return <div>Register</div>;
+  const [customers, setCustomers] = useState([]);
+  const [customer, setCustomer] = useState({
+    firstname: "",
+    lastname: "",
+    driverLicense: "",
+    email: "",
+    password: "",
+    phoneNumber: "",
+    dob: "",
+    goldMembership: false,
+    province: "",
+    city: "",
+    postalCode: "",
+    streetNumber: "",
+    streetName: "",
+    unitNumber: "",
+  });
+
+  const handleChange = (e) => {
+    setCustomer({ ...customer, [e.target.name]: e.target.value });
+  };
+  const getCustomers = () => {
```

Comment:
I don't think there's a need to get all customers for this page I really don't see a reason to do so as you're just registering a customer. when registering someone it doesn't mean that you need to use a state then add it to the state, I only do that when I need to use all the customers and I want to show the new added customer to them without a refresh